src/androidTest/java/com/mixpanel/android/mpmetrics/DecideCheckerTest.java

```diff
@@ -191,18 +195,24 @@ public void testBadDecideResponses() throws ServiceUnavailableException {
         mEventBinder.bindingsSeen.clear();
     }
 
-    public void testDecideHonorsFallbackDisabled() throws ServiceUnavailableException {
+    public void testDecideHonorsFallbackEnabled() throws RemoteService.ServiceUnavailableException {
         mConfig.fallbackDisabled = false;
-        mPoster.response = bytes("{\"surveys\":[], \"notifications\":[]}");
+        mPoster.requestedUrls.clear();
+        mPoster.response = null;
+        mPoster.exception = new IOException("Bang!");
         mDecideChecker.addDecideCheck(mDecideMessages1);
         mDecideChecker.runDecideChecks(mPoster);
-        assertEquals(mPoster.requestedUrls.length, 2);
+        assertEquals(2, mPoster.requestedUrls.size());
```

Comment:
YES! This is the proper order of parameters for assertEquals. And we are having wrong orders all over the place.